neo/d3xp/script/Script_Interpreter.cpp

```diff
@@ -176,135 +178,166 @@ void idInterpreter::Reset( void ) {
 	doneProcessing	= true;
 }
 
-/*
-================
-idInterpreter::GetRegisterValue
-
-Returns a string representation of the value of the register.  This is
-used primarily for the debugger and debugging
-
-//FIXME:  This is pretty much wrong.  won't access data in most situations.
-================
-*/
-bool idInterpreter::GetRegisterValue( const char *name, idStr &out, int scopeDepth ) {
+bool idInterpreter::GetRegisterValue(const char* name, idStr& out, int scopeDepth) {
```

Comment:
unnecessary (but at least consistent) whitespace change; might be preferable to fix the doc comment instead of deleting it

Reply:
yes will fix comment!